### DISCUSSION TITLE
Save sign in and server URL in Android app?

### DISCUSSION THREAD

**Discussion by starbuck93** [immich-app/immich#122] [General]
Hello! I just set up the server v1.6.0 with docker and the Android app from the Play store. Everything went smoothly! The Android app seems to work well, but I noticed that I have to sign in and type in the server URL every time I close out of the app. (only when I swipe away the app from Recents, if that makes sense.) It would be nice to have a "save information" option or a settings menu buried under the login page to save these resources. Thanks!

**Comment by alextran1502** [comment]
Hello,

There is an issue open #96 for this request :) I will be working on this on the next release.
